collections/_article/dev-snapshot-godot-4-4-dev-6.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Dev snapshot: Godot 4.4 dev 6"
+excerpt: "Featuring: Malware! (does not contain malware)"
+categories: ["pre-release"]
+author: Thaddeus Crews
+image: /storage/blog/covers/dev-snapshot-godot-4-4-dev-6.webp
+image_caption_title: "Malware"
+image_caption_description: "A game by OddGames"
+date: 2024-12-04 12:00:00
+---
+
+Safety goggles on, people! Features and Quality-of-Life are being integrated at blinding speeds; precautions must be taken to view them head-on. Once secure, we can dive into what's shaping up to be our final development snapshot before 4.4-beta builds roll out!
+
+Many of the changes in this release are bug fixes that will be backported to Godot 4.3 and released in 4.3.1! So please
+test this release well so we can be confident with the changes and release 4.3.1 with them as soon as possible.
+
+Keep in mind that while we try to make sure each dev snapshot is stable enough for general testing, this is by
+definition a pre-release piece of software. Be sure to make frequent backups, or use a version control system such as
+Git, to preserve your projects in case of corruption or data loss.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also try the [**Web editor**](https://editor.godotengine.org/releases/4.4.dev6/), [**XR editor**](https://www.meta.com/experiences/godot-game-engine/7713660705416473/), or the **Android editor** for this release (join the [Android editor testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds).
+
+-----
+
+*The cover illustration is from* [**Malware**](https://store.steampowered.com/app/3019370/Malware/), *an installation simulator where players must navigate a minefield of circa-1999 maleware! It is developed by [OddGames](https://store.steampowered.com/search/?developer=Odd%20Games). You can purchase the game [on Steam](https://store.steampowered.com/app/3019370/Malware/).*
+
+## Highlights
+
+In case you missed them, see the [4.4 dev 1](/article/dev-snapshot-godot-4-4-dev-1/), [4.4 dev 2](/article/dev-snapshot-godot-4-4-dev-2/),
+[4.4 dev 3](/article/dev-snapshot-godot-4-4-dev-3/), [4.4 dev 4](/article/dev-snapshot-godot-4-4-dev-4/), and [4.4 dev 5](/article/dev-snapshot-godot-4-4-dev-5/) release notes for an overview of
+some key features which were already in that snapshot, and are therefore still available for testing in dev 6.
+
+Here are highlights of a few new features in dev 6 that you might find particularly exciting!
+
+### Camera3D preview in inspector
+
+Words words words
+
+<video autoplay loop muted playsinline>
+  <source src="/storage/blog/dev-snapshot-godot-4-4-dev-6/camera3d-inspector.mp4?1" type="video/mp4">
+</video>
+
+### CollisionShape3D debug color customization
+
+Words words words
+
+![Examples of custom debug colors](/storage/blog/dev-snapshot-godot-4-4-dev-6/debug-color.webp)
+
+### And more!
```

Comment:
I'm plugging my own feature here, but I think it warrants a preview. Feel free to disagree.

````suggestion
### Temporary file and directory utilities

Tool creators, here's something that could be interesting for you. A brand new API let's you create and dispose of temporary files. Handy if you need to write content on disk, but without worrying about polluting user data. These temporary files and directories even dispose of themselves after use by default. For more information, see ([GH-98397](https://github.com/godotengine/godot/pull/98397)).

### And more!
````